packages/oas-utils/src/transforms/import-spec.ts

```diff
@@ -410,25 +410,6 @@ export async function importSpecToWorkspace(
         delete requestPayload.examples
       }
 
-      // Add list of UIDs to associate security schemes
```

Comment:
as far as I can see, all this code seems to do is break complex auth

Reply:
lol